postcss.config.js

```diff
@@ -0,0 +1,11 @@
+const postcss = require('postcss');
```

Comment:
How about moving all the default config into this file? Then it could easily be reused by users with `postcss-cli`.